frameworks/g-cloud-12/questions/services/standardsISO28000When.yml

```diff
@@ -13,6 +13,6 @@ type: text
 
 validations:
   - name: answer_required
-    message: You need to answer this question.
+    message: Enter the accreditation date.
   - name: under_character_limit
```

Comment:
Query: should the entry of the date be in pre-set format, and thereby making a character limit redundant?

Reply:
See above